README.md

```diff
@@ -0,0 +1,39 @@
+# GraphQL ToDo Server
+
+A basic GraphQL API for a ToDo application.
+
+## Clone
+
+```bash
+git clone git@github.com:prmsrswt/graphql-todo-server.git
+```
+
+### Install Dependencies
+
+**Yarn**
+
+```bash
+yarn
+```
+
+**NPM**
+
+```bash
+npm install
+```
```

Comment:
This would seem like we need to do both `yarn` and `npm install`. Change this to something like
```
```bash
yarn # or if you use npm then, npm install
\```
```

Reply:
Done.